packages/commons/utils/commons.ts

```diff
@@ -0,0 +1,55 @@
+import { Transaction } from 'ethers';
+import { encode } from 'rlp';
+
+export const normalizeAddress = (address: string): string => address.toLowerCase();
+
+// Ref: https://eips.ethereum.org/EIPS/eip-2718, https://eips.ethereum.org/EIPS/eip-2930 and https://eips.ethereum.org/EIPS/eip-1559
+export const getRawTransaction = (transaction: Transaction): string => {
```

Comment:
This took some time to get right. I was able to test it with type == 2 and type == 0, was not able to test with type == 1.

Reply:
oh my, nice job